executor/runtime/types/types.go

```diff
@@ -51,6 +52,8 @@ const (
 	testImageTag     = "latest"
 )
 
+var Version = ""
```

Comment:
can you make this `titusExecutorVersion` ?

Is there any other place we can put this where it can be more accessible? Where it would be easy to put it in logs or help text or other places? (types.go seems like a strange place to put it. Maybe /executor/version.go ?)

Reply:
Sure, I was trying to preemptively YAGNI this, but I can move it there.

Reply:
I know, it is a balancing act. I swear we will use this :)